HealthcareSystem/PatientWebApp/Controllers/SurveyController.cs

```diff
@@ -46,60 +37,28 @@ public ActionResult GetPermissions()
         {
             var patientJmbg = HttpContext.User.FindFirst("Jmbg").Value;
 
-            var client = new RestClient(_serviceSettings.FeedbackAndSurveyServiceUrl);
-            var request = new RestRequest("/api/survey/patient/" + patientJmbg + "/permission", Method.GET);
-            var response = client.Execute(request);
-
-            var contentResult = new ContentResult();
-            contentResult.Content = response.Content;
-            contentResult.ContentType = "application/json";
-            contentResult.StatusCode = (int)response.StatusCode;
-            return contentResult;
+            return RequestAdapter.SendRequestWithoutBody(_serviceSettings.FeedbackAndSurveyServiceUrl, "/api/survey/patient/" + patientJmbg + "/permission", Method.GET);
```

Comment:
One tiny suggestion I have is to put each parameter into a new line in these calls, since the parameter list gets really long sometimes.

Reply:
I'll refactor that for the next sprint.